### PR TITLE
feat: Initialization docker 

### PR DESCRIPTION
**Pull request by sumit-158**
### What
- Initializing docker-compose (currently django )
### Screenshot
docker

### Problem 
- The backend(django) working properly , but the flutter web(forntend) exit with error you can see in the screenshot
### Fixes bug(s)
- #10

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    build: /community_portal
+    command: bash -c "python community_portal/manage.py makemigrations && python community_portal/manage.py migrate && python community_portal/manage.py runserver 0.0.0.0:8000"
+    container_name: community_portal
+    volumes:
+      - .:/community_portal
+    ports:
+      - "8000:8000"
+  client:
+    build: /client
+    command: flutter run -d chrome
```
Comment (aquilax) [inline docker-compose.yml:14]: There is no flutter command in the container. It contains only the built assets.

Comment (aquilax) [reply docker-compose.yml]: Try something like that:
```
version: '3'

services:
  server:
    build: ./community_portal
    command: bash -c "python community_portal/manage.py makemigrations && python community_portal/manage.py migrate && python community_portal/manage.py runserver 0.0.0.0:8000"
    container_name: community_portal
    volumes:
      - .:/community_portal
    ports:
      - "8000:8000"
  client:
    build: ./client
    container_name: client
    volumes:
      - .:/client
    ports:
      - "8001:80"
    environment:
      - NGINX_PORT=80
```

Comment (sumit-158) [reply docker-compose.yml]: image
look!

Comment (sumit-158) [reply docker-compose.yml]: Thank you so much ! :)

**Review by aquilax — APPROVED** [review]

**Pull request merged**